python/ct/client/log_client.py

```diff
@@ -196,8 +196,9 @@ def _parse_consistency_proof(response, servername):
 # A class that we can mock out to generate fake responses.
 class RequestHandler(object):
     """HTTPS requests."""
-    def __init__(self, connection_timeout=60):
+    def __init__(self, connection_timeout=60, ca_bundle=True):
```

Comment:
The ca_bundle is being passed into verify, which can take one of the following values:
True: SSL certificate validation will take place using the default CA root certs set.
False: SSL certificate validation will not take place.
Path to a file: SSL certificate validation will take place using the CA root set specified in this file.

See requests documentation:
http://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification